diag_manager/fms_diag_object.F90

```diff
@@ -67,8 +67,8 @@ module fms_diag_object_mod
      class(*), allocatable, private                   :: missing_value     !< The missing fill value
      class(*), allocatable, private                   :: data_RANGE        !< The range of the variable data
      type (diag_axis_type), allocatable, dimension(:) :: axis              !< The axis object
-!> \brief Extends the variable object to work with multiple types of data
-     class(*), allocatable :: vardata0
+! data buffers
+     class(*), allocatable :: vardata0                                     
```

Comment:
This is adding leading whitespace

Reply:
This was just all wrong anyway.  The data buffers now all have their own comments.